soc/xtensa/nxp_adsp/imx8m/Kconfig.defconfig.series

```diff
@@ -29,6 +29,9 @@ config SYS_CLOCK_HW_CYCLES_PER_SEC
 config SYS_CLOCK_TICKS_PER_SEC
 	default 50000
 
+config IPM_IMX_REV2
```

Comment:
Is this needed? I expect that `IPM_IMX_REV2` should be enabled when `mailbox0` is `status=okay` and `CONFIG_IPM=y`